src/Essentials/src/SecureStorage/SecureStorage.ios.tvos.watchos.macos.cs

```diff
@@ -9,8 +9,24 @@ namespace Microsoft.Maui.Essentials
 {
 	public static partial class SecureStorage
 	{
-		public static SecAccessible DefaultAccessible { get; set; } =
-			SecAccessible.AfterFirstUnlock;
+		public static SecAccessible DefaultAccessible
+		{
+			get { return Implementations.SecureStorageImplementation.DefaultAccessible; }
+			set { Implementations.SecureStorageImplementation.DefaultAccessible = value; }
+		}
+
+		public static Task SetAsync(string key, string value, SecAccessible accessible) =>
+			Implementations.SecureStorageImplementation.SetAsync(key, value, accessible);
+	}
+}
+
+namespace Microsoft.Maui.Essentials.Implementations
```

Comment:
Here and in #4630 I had to make such namespace splits, since the static class doesn't live in the `Implementations` namespace. It seemed unworthy to split up the files, but if this is an unprecedented code pattern maybe I should do it?

@Redth Thoughts?

Reply:
I think it's fine in this case to have both in the same file :)